<html>
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <Meta http-equiv="Content-Style-Type" content="text/css">
  <title>Deep Sequencing</title>
  <style type="text/css">
  body {
   font-family: "PT serif", Georgia, serif;
   font-size: 1.2em;
   padding: 60px;
   width: 800px;  
  }
  code {
   font-family: Consolas, Monaco, Courier;
   font-size: 14px;
   padding-left: 0px;
   background: #ffff99;
  }  
  li {
    margin-bottom: 1em;
  }
  </style>
</head>
<body>

<h1><i>Deep Sequencing</i></h1> 

<h3>G4017</h3>


<h4>Offered in Fall 2015</h4>

<h4>Credits: 3</h4>

<h3>Schedule:</h3>

<p>Class is held every Tuesday 2:00pm-4:00pm starting 9/8/2015 in ICRC 817. <br /> 

There will be one two-hour lecture per week (12 lectures), a take-home midterm exam will be distributed in Week 8, and final project presentations will be delivered in the last two weeks. </p>

<h3>Instructors:</h3>

<ul>
<li>Dr. Yufeng Shen, ys2411@columbia.edu, ICRC 812A</li>
<li>Dr. Peter Sims, pas2182@columbia.edu, Lasker 207</li>
<li>Dr. Chaolin Zhang, cz2294@columbia.edu, P&amp;S 4-448</li>
</ul>

<p>Instructors will be available for office hours by appointment.  In addition, the instructor delivering the weekly Tuesday lecture will be available for office hours that same week on Friday afternoon (1:30pm-3:00pm).</p>

<h3>Overview:</h3>

<p>Next-generation sequencing (NGS) has become ubiquitous in biomedical research with numerous applications. This course will provide an in-depth introduction to principles of modern sequencing, key computational algorithms and statistical models, and applications in disease genetics, cancer and fundamental biology. It will cover genome, exome and transcriptome sequencing approaches.  Emphasis will be placed on understanding the interplay between experimental design, data acquisition, and data analysis so that students can apply these powerful tools in their own research.</p>

<h3>Topics:</h3>

<ul>
<li>History and development of modern sequencing technologies</li>
<li>Introduction to statistics and algorithms</li>
<li>Genome sequencing and genetics of Mendelian and complex diseases (analysis and laboratory techniques for whole genome and exome sequencing)</li>
<li>RNA-Seq and transcriptional regulation (analysis and laboratory techniques for expression profiling, CLIP-Seq, ribosome profiling, micro-RNA profiling).</li>
<li>Cancer genomics</li>
<li>Emerging third and fourth generation sequencing technologies and single cell analysis.</li>
</ul>

<h3>Goals:</h3>

<ul>
<li>To acquire introductory knowledge of modern high-throughput sequencing approaches including instrumentation and laboratory techniques.</li>
<li>To understand computational methods for analyzing genome sequencing data.</li>
<li>To formulate a biological question and investigate it by analyzing existing sequencing data. </li>
</ul>

<h3>Requirements:</h3>
<p>This course is intended for graduate students or senior undergraduates interested in learning state-of-the-art sequencing approaches and their applications in biological and medical research. </p>

<p><strong>Required:</strong> Basic knowledge in molecular biology, probability, and statistics</p>
<p><strong>Preferred:</strong> Working knowledge of UNIX and a programming language (R, MATLAB, Python, or Perl). Basic knowledge of genetics would also be helpful.</p>

<h3>Grading:</h3>

<ul>
<li>Class Participation: 10%</li>
<li>Journal Club Presentation: 20%</li>
<li>Midterm Exam: 20%</li>
<li>Final Project: 50%</li>
</ul>

<p>The course will be highly interactive and require class participation on which each student will be graded (10% of course grade). Throughout the course, there will be ten journal club presentations on pre-assigned papers.  Teams of 2-3 students will be randomly assigned to a week/paper during Week 1 for 30-40 minute journal club presentations (20% of course grade). There will be a take-home midterm exam in Week 8 (20% of course grade). Students will assemble into teams with 3-4 members to pursue a Final Project as a group (50% of course grade). Abstracts will be presented in Week 8. Each team will produce a paper describing their results and deliver a final oral presentation in the last two weeks of the course.</p>

<h3>Weekly schedule and reading materials</h3>

<ul>
<li>Week 1 (9/8, Sims): Introduction to High-Throughput Sequencing Technologies.</li>
</ul>

<p>Reading: (1) Sanger et al., DNA sequencing with chain-terminating inhibitors. PNAS, 74, 5463-5467 (1977). (2) Rothberg et al. Genome sequencing in microfabricated high-density picoliter reactors. Nature, 437, 376-380 (2005). (3) Shendure et al. Accurate multiplex polony sequencing of an evolved bacterial genome. Science, 309, 1728-1732 (2005). (4) Bentley et al., Accurate whole genome sequencing using reversible terminator chemistry. Nature, 456, 53-59 (2008).</p>

<p>Journal Club: None.</p>

<ul>
<li>Week 2 (9/15, Sims): Experimental Techniques for Genomics. </li>
</ul>

<p>Reading: (1) Venter et al., The sequence of the human genome. Science, 291, 1304-1351 (2001). (2) Wheeler et al., The complete genome of an individual by massively parallel DNA sequencing. Nature, 452, 872-876 (2008). (3) Porreca et al., Multiplex amplification of large sets of human exons. Nature Methods, 4, 931-936 (2007). (4) Mamanova et al., Target-enrichment strategies for next-generation sequencing. Nature Methods, 7, 111-118 (2010). (5) Adey et al., Rapid, low-input, low-bias construction of shotgun fragment libraries by high-density in vitro transposition. Genome Biology, 11, R119 (2010).</p>

<p>Journal Club: Amini et al. Haplotype-resolved whole-genome sequencing by contiguity-preserving transposition and combinatorial indexing. Nature Genetics, 46, 1343-1349 (2014).</p>

<ul>
<li>Week 3 (9/22, Sims): Transcriptome Sequencing Technologies.</li>
</ul>

<p>Reading: (1) Mortazavi et al., Mapping and quantifying mammalian transcriptomes by RNA-Seq. Nature Methods, 5, 621-628 (2008). (2) Licatalosi et al., HITS-CLIP yields genome-wide insights into brain alternative RNA processing. Nature, 456, 464-469 (2008). (3) Ingolia et al., Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling. Science, 324, 218-223 (2009). (4) Shiroguchi et al., Digital RNA sequencing minimizes sequence-dependent bias and amplification noise with optimized single-molecule barcodes. PNAS, 109, 1347-1352 (2012). (5) Rouskin, S., Zubradt, M., Washietl, S., Kellis, M. &amp; Weissman, J.S. Genome-wide probing of RNA structure reveals active unfolding of mRNA structures in vivo. Nature 505, 701 (2014).</p>

<p>Journal Club: Jan, C.H., Williams, C.C., Weissman, J.S. Principles of ER cotranslational translocation revealed by proximity-specific ribosome profiling. Science, 346, 1257521 (2014) along with Reid, D.W., Nicchitta, C.V. Comment on: “Principles of ER cotranslational translocation revealed by proximity-specific ribosome profiling.” Science, 348, 1217 (2015) and response to comment.</p>

<ul>
<li>Week 4 (9/29, Shen/Zhang/Sims): Project Ideas, Team Forming, Computational Bootcamp.</li>
</ul>

<p>Reading: Web-based resources will be available.
Teams of 3-4 students with balanced expertise will form for Final Projects. Instructors will propose a list of suggested topics for Final Projects, and teams can either select one or develop their own.  Instructors will conduct an optional computational bootcamp to provide basic instruction in using the UNIX operating system, scripting, and analyzing a simple data set with R.</p>

<ul>
<li>Week 5 (10/6, Zhang): Transcriptome Sequencing Technologies (continued).</li>
</ul>

<p>Journal Club: Rouskin, S., Zubradt, M., Washietl, S., Kellis, M. &amp; Weissman, J.S. Genome-wide probing of RNA structure reveals active unfolding of mRNA structures in vivo. Nature 505, 701 (2014).</p>

<ul>
<li>Week 6 (10/13, Zhang): Statistical Models for Sequencing Data.</li>
</ul>

<p>Reading: Biological Sequence Analysis: Probabilistic Models of Proteins and Nucleic Acids, by Durbin, Eddy et al. </p>

<ul>
<li>Week 7 (10/20, Shen): Statistical Models for Sequencing Data (continued).</li>
</ul>

<p>Journal Club: PICK TWO 1) Eddy, S.R. What is a hidden Markov model? Nature Biotechnology, PMID: 15470472 (2004). 2) Krzywinski, M., Altman, N. Power and sample size. Nature Methods (2013). 3) Noble, W.S. How does multiple testing correction work? Nature Biotechnology, PMID: 20010596 (2009).</p>

<ul>
<li>Week 8 (10/27, Shen): Genome Sequencing and Algorithms.</li>
</ul>

<p>Following a one hour lecture, each Final Project team will submit a written abstract and present a five-minute summary of their plans. A take-home midterm exam will be distributed.  The midterm exam must be completed and returned at the beginning of class in Week 9 (11/3 at 2:00 pm).</p>

<p>Reading: (1) A framework for variation discovery and genotyping using next-generation DNA sequencing data, by DePristo et al 2011, Nature Genetics, doi: 10.1038/ng.806. PMID: 21478889. (2) Fast and accurate short read alignment with Burrows-Wheeler transform, by Li and Durbin, 2009, Bioinformatics, doi: 10.1093/bioinformatics/btp324. PMID: 19451168. (3) Exome sequencing identifies the cause of a mendelian disorder, by Ng et al 2010, Nature Genetics, doi: 10.1038/ng.499. PMID: 19915526. (4) DeNovoGear: de novo indel and point mutation discovery and phasing, by Ramu et al 2013, Nature Methods, doi: 10.1038/nmeth.2611. PMID: 23975140. (5) Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples, by Cibulskis et al 2013, Nature Biotech, doi: 10.1038/nbt.2514. PMID: 23396013. (6) A map of human genome variation from population-scale sequencing, by Abecasis et al 2010, Nature, doi: 10.1038/nature09534. PMID: 20981092.</p>

<ul>
<li>Week 9 (11/3, Shen): Genome Sequencing and Algorithms (continued).</li>
</ul>

<p>Take-home midterm exam is due at the beginning of class (2:00 pm).</p>

<p>Journal Club: PICK ONE 1) Cibulskis et al. Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples. Nature Biotechnology, PMID: 23396013 (2013). 2) DePristo et al. A framework for variation discovery and genotyping using next-generation DNA sequencing data. Nature Genetics, PMID: 21478889, (2011).</p>

<ul>
<li>Week 10 (11/10, Shen): Genome Sequencing and Applications in Genetics Studies of Human Diseases.  </li>
</ul>

<p>Reading:   (1) Synaptic, transcriptional and chromatin genes disrupted in autism, by De Rubeis et al 2014, Nature, doi: 10.1038/nature13772. PMID: 25363760.  (2) DNA sequencing of a cytogenetically normal acute myeloid leukaemia genome, by Ley et al, 2008, Nature, doi: 10.1038/nature07485. PMID: 18987736. (3) Comprehensive molecular profiling of lung adenocarcinoma, by TCGA, 2014, Nature, doi: 10.1038/nature13385. PMID: 25079552. (4) Large-scale discovery of novel genetic causes of developmental disorders, by The DDD Study, 2014, Nature, doi: 10.1038/nature14135. PMID: 25533962</p>

<p>Journal Club: De Rubeis et al. Synaptic, transcriptional and chromatin genes disrupted in autism. Nature, PMID 25363760 (2014).</p>

<ul>
<li>Week 11 (11/17, Zhang): Transcriptome Sequencing: Algorithms and Applications in Studying Regulation of Gene Expression.     </li>
</ul>

<p>Reading: (1) Wu, J., Anczukow, O., Krainer, A.R., Zhang, M.Q. &amp; Zhang, C. OLego: Fast and sensitive mapping of spliced mRNA-Seq reads using small seeds. Nucleic Acids Res 41, 5149-5163 (2013).  (2) Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29, 15-21 (2013). (3) Anders, S. et al. Count-based differential expression analysis of RNA sequencing data using R and Bioconductor. Nature Protocols 8, 1765-1786 (2013). (4) Shen, S. et al. rMATS: Robust and flexible detection of differential alternative splicing from replicate RNA-Seq data. Proc Natl Acad Sci U S A 111, E5593-601 (2014). (5) Zhang, C. &amp; Darnell, R.B. Mapping in vivo protein-RNA interactions at single-nucleotide resolution from HITS-CLIP data. Nat Biotech 29, 607-614 (2011).</p>

<p>Journal Club: Hiller, D., Jiang, H., Xu, W., Wong, W.H. Identifiability of isoform deconvolution from junction arrays and RNA-Seq. Bioinformatics, 25, 3056-3059 (2009).</p>

<ul>
<li>Week 12 (11/24, Zhang): Transcriptome Sequencing: Algorithms and Applications in Studying Regulation of Gene Expression (continued).</li>
</ul>

<p>Journal Club: Li, M. et al. Widespread RNA and DNA sequence difference in the human transcriptome. Science, 333, 53-58 (2011). Lin, W., Piskol, R., Tan, M.H., Li, J.B. Comment on “Widespread RNA and DNA sequence difference in the human transcriptome”. Science, 335, 1302; with author reply (2012).</p>

<ul>
<li>Week 13  (12/1, Sims): Emerging Technology for Sequencing and Single Cell Analysis. </li>
</ul>

<p>Reading: (1) Brock et al., Accurate whole-genome sequencing and haplotyping from 10 to 20 human cells. Nature, 487, 190-195 (2012). (2) Laszlo et al., Decoding long nanopore sequencing reads of natural DNA. Nature Biotechnology, 32, 829-833 (2014). (3) Tang et al., mRNA-Seq whole-transcriptome analysis of a single cell. Nature Methods, 6, 377-382 (2009). (4) Pollen et al., Low-coverage single-cell mRNA sequencing reveals cellular heterogeneity and activated signaling pathways in developing cerebral cortex. Nature Biotechnology, 32, 1053-1058 (2014). (5) Zong et al., Genome-wide detection of single-nucleotide and copy-number variations of a single human cell. Science, 338, 1622-1626 (2012).</p>

<p>Journal Club: McConnell, M.J. et al. Mosaic copy number variation in human neurons. Science, 342, 632-637 (2013).</p>

<ul>
<li>Week 14 (12/8, Shen/Zhang/Sims): Final Project Presentations.</li>
<li>Week 15 (12/15, Shen/Zhang/Sims): Final Project Presentations (continued).</li>
</ul>





 <h4><a href="http://www.columbia.edu/~ys2411/">Shen Lab</a></h4>

<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-400081-8', 'columbia.edu');
  ga('send', 'pageview');
</script>
</body>

</html>


